install.md

```diff
@@ -17,7 +17,15 @@ Select the operating system you are working on.
 
 <div id="tab-osx" class="content active">
 
-[Node.js](http://nodejs.org/) is a prerequisite for installing the Tessel command line. You can install it from the [official website](http://nodejs.org/), or if you have [brew](http://brew.sh/) installed: `brew install node`
+[Node.js](http://nodejs.org/) is a prerequisite for installing the Tessel command line. You can install it using nvm by running the following commands in your terminal:
+
+`curl https://raw.githubusercontent.com/creationix/nvm/v0.23.2/install.sh | bash`
+
+`source ~/.bashrc`
+
+`nvm install 0.12.7`
```

Comment:
> @Student007 curious to get your take as well

@Frijol sorry - I am out of time today - I assume I will not be free next weekdays.

The best way to verify is to use [my reference machines](https://github.com/Student007/t2-reference-VMs) - that way you will also see some traps. You need to rewrite/cleanup the vagrant file to execute `curl https://raw.githubusercontent.com/creationix/nvm/v0.23.2/install.sh | bash` and following steps.
Note: The MacOS X reference machine needs up to 20 hours to download the image. But after done once you can work with it like the Windows- and Linux- VMs.

Hope my hard work at this reference images will be payed by being useful at this point :+1: